build.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+from __future__ import print_function, division
+import os
+import sys
+from base64 import b64decode
+
+import requests
+from tqdm import tqdm
+from github import Github
+import yaml
+
+g = Github(os.getenv("GITHUB_API_TOKEN"))
+repos = list(g.get_organization("OpenWorm").get_repos())
+
+
+def repo2content(repo):
+    """
+    @return {"filename": "content"} where `filename` starts with `.openworm.`
+    """
+    tree = repo.get_git_tree("master", recursive=False).tree
+    files = [i for i in tree if i.path.lower().startswith('.openworm.')]
+    if not files:
+        return {}
+    reqs = [requests.get(i.url) for i in files]
+    try:
+        data = [b64decode(i.json()['content']) for i in reqs]
+    except KeyError:
+        raise ValueError(reqs)  # likely 403
+    return dict(zip((i.path.lower() for i in files), data))
+
+
+def repo2meta(repo):
+    content = repo2content(repo)
+    if not content:
+        return {}
+    meta = {}
+    for file, data in content.items():
+        if file.endswith('.yml') or file.endswith('.yaml'):
+            meta.update(yaml.safe_load(data))
+        elif file.endswith('.rst'):
```

Comment:
I have been wondering about this, so I'll ask here: what's the need to support multiple file types? Adding options adds complexity, but I don't see the utility: There's no prior work to integrate, so you can just pick the easiest format to work with (in your assessment).